README.md

```diff
@@ -60,7 +60,19 @@ listening for API requests and provisioning volumes accordingly.
 
 ### Architecture and Operation
 
-The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct
+The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct. This tables describes shows overview and comparison of those modes. There is more detailed modes explanation down in following paragraphs.
+
+|                   |LVM                    |Direct              |
+|:--                |:--                    |:--                 |
+|Purpose            |avoids fragmentation   |simpler, somewhat faster, but storage may get fragmented   |
```

Comment:
"Purpose" -> "Main advantage"?

"fragmentation" is not obvious. Add a footnote that explains it?

Reply:
should we explain this by text we make up, or point to generic definition like https://en.wikipedia.org/wiki/Fragmentation_(computing)#Data_fragmentation

Reply:
I would provide a short example.

Reply:
Users are probably aware what "fragmentation" means, just not how it applies in this context.